Classes/Controller/AbstractActionController.php

```diff
@@ -34,7 +34,7 @@
      * Needed in TYPO3 6.0.0-6.0.99
      *
      * @var \TYPO3\CMS\Extbase\SignalSlot\Dispatcher
-     * @inject
+     * @TYPO3\CMS\Extbase\Annotation\Inject
```

Comment:
This one is fine for TYPO3 v9 but won't work in TYPO3 v8: https://docs.typo3.org/typo3cms/extensions/core/Changelog/9.0/Deprecation-82869-ReplaceInjectWithTYPO3CMSExtbaseAnnotationInject.html